src/GDGUkraine/lib/utils/table_exporter.py

```diff
@@ -1,8 +1,41 @@
 from io import BytesIO
+from json import dumps as json_dumps
+
 from openpyxl import Workbook
 from openpyxl.writer.excel import save_virtual_workbook
 
 
+event_parsers = (
```

Comment:
This is used only once (as for test I will write there too). Move to `gen_participants_xlsx`